docs/src/user/installing.md

```diff
@@ -12,30 +12,39 @@ rustup target add --toolchain stable wasm32-wasi
 
 ## Installing from Git
 
-Ambient can be installed through `cargo install`, which will download and build the repository. Our current minimum supported Rust version is <!-- rust-version-begin !-->1.67.0<!-- rust-version-end !-->, as we use recently-stabilised standard library features.
+*Ambient* can be installed through `cargo install` using git.
 
-To install the latest released version from Git, run the following:
+This will download the source and compile the runtime. Our minimum supported Rust version is <!-- rust-version-begin -->1.67.0<!-- rust-version-end -->.
+
+### Installing the latest published release
+
+This is recommended if you want a binary for your platform using the last stable release
```

Comment:
"This is the recommended method of installing Ambient from source if the downloadable binaries are insufficient. The latest published release should be used unless you have a specific reason to use the development version."